conanfile.py

```diff
@@ -674,6 +674,34 @@ def package_id(self):
     def package_info(self):
         self.env_info.CMAKE_PREFIX_PATH.append(self.package_folder)
 
+        self.cpp_info.libs = tools.collect_libs(self)
+
+        # "Qt5Bootstrap.lib" contains symbols that are also in "Qt5Core.lib",
+        # removing it so linker succeed.
+        if 'Qt5Bootstrap' in self.cpp_info.libs:
+            self.cpp_info.libs.remove('Qt5Bootstrap')
```

Comment:
if we never link to Qt5Bootstrap lib, maybe it's worth deleting it in package(self) ? Do you know why qt installs it?

Reply:
I have no idea what is `Qt5Bootstrap` purpose.
Qt expert opinion would be very helpful here.

IMO if we remove `Qt5Bootstrap` from package and turns out that it is needed in some cases, than package became useless. If we leave it, and only remove from linking, then this package can used and linking may be fixed in consumer code (maybe simple `target_link_libraries(MyProgram, Qt5Bootstrap)` will work)

Reply:
I just got some information that Qt5Bootstrap must not be used by any other project that qt itself, so it's safe to remove the file in package(self)

Reply:
Done